### DISCUSSION TITLE
Github Campus Experts

### DISCUSSION THREAD

**Discussion by toukirkhan** [community/community#33668] [GitHub Education] [answered]
Hi everyone, I have applied twice for the github campus expert till now  and both the time I did not got the acceptance mail nor the rejection can anyone help me regarding the same.

**Comment by cosmicqbit** [comment, marked as answer]
Github is still sending the responses out. May be wait more for a couple of days. I have also applied for Campus Expert Program and Github started sending responses from 19 Sep & I didn't got any mail back either yet. If you don't get the response within next week you can take this to twitter or submit a ticket on [Github Support](https://support.github.com/request) Keep check on your mailbox and the social handle of [Github Education on twitter ](https://twitter.com/GitHubEducation) for updates.

**Reply by txaty** [reply]
I haven't received the result either. I have changed my username from tommytim0515 to txaty after the application, and probably it is the reason. When I log in to the campus expert webpage, it shows that I haven't registered yet. I also created a ticket to GitHub about this but no one responded yet.

**Reply by pranayughade** [reply]
Same here

**Comment by toukirkhan** [comment]
Almost all of my friends at my campus got the result for the application
but I didn't got for the August 2022. And also I haven't got any email for
the FEBRUARY 2022 APPLICATION too. Please consider.

**Reply by cosmicqbit** [reply]
You can track your application status [here](https://apply.githubcampus.expert/auth/github) or you can also tweet about this and tag @Githubeducation

**Comment by toukirkhan** [comment]
It doesn't works

**Comment by toukirkhan** [comment]
Please understand I didn't got mail even for the February 2022 application.

**Comment by cosmicqbit** [comment]


**Reply by notarib-catcher** [reply]
Adding onto this, The campus expert site seems to error out when I try to check my application status. 😔

**Reply by cosmicqbit** [reply]
You ain't alone man. The status link is down for everyone.

**Comment by toukirkhan** [comment]
Okay got it

**Comment by toukirkhan** [comment]
Yeah, can you connect with me on twitter?

**Reply by cosmicqbit** [reply]
Sure, my ID is [@cosmicqbit ](https://twitter.com/cosmicqbit)

**Comment by toukirkhan** [comment]
Okay cool mine is @_toukirkhan_

**Comment by toukirkhan** [comment]
Cool!
